cloudformation/apispec.yaml

```diff
@@ -0,0 +1,58 @@
+---
+openapi: 3.0.3
+info:
+  title: Xplorers API
+  version: 0.0.1
+  license:
+    name: MIT
+    url: https://opensource.org/licenses/MIT
+paths:
+  /:
+    post:
+      responses:
+          200:
+            description: 200 response
+            content:
+              application/json:
+                schema:
+                  $ref: "#/components/schemas/Empty"
+      x-amazon-apigateway-integration:
+        type: aws_proxy
+        httpMethod: POST
+        responses:
+          default:
+            description: XplorersBot Response
+            statusCode: 200
+        passthroughBehavior : when_no_templates
+        contentHandling : CONVERT_TO_TEXT
+        uri:
+          Fn::Sub: arn:${AWS::Partition}:apigateway:${AWS::Region}:lambda:path/2015-03-31/functions/arn:${AWS::Partition}:lambda:${AWS::Region}:${AWS::AccountId}:function:${XplorersBotGo}/invocations
+  /chatbot:
+    post:
+      responses:
+          200:
+            description: 200 response
+            content:
+              application/json:
+                schema:
+                  $ref: "#/components/schemas/Empty"
+      x-amazon-apigateway-integration:
+        httpMethod: POST
+        responses:
+          default:
+            description: Xplorers ChatBot Response
+            statusCode: 200
```

Comment:
Seme as above. Only use `200` for when you request was completed and no further actions is to be done.